tslint.json

```diff
@@ -4,6 +4,7 @@
         "max-line-length": false,
         "no-empty": false,
         "no-shadowed-variable": false,
-        "object-literal-sort-keys": false
+        "object-literal-sort-keys": false,
+        "no-console": false
```

Comment:
Hey hey, this seems unnecessary.

Reply:
good catch, I added this to deploy and log in integration 🤦